src/Raven.Server/Web/System/OngoingTasksHandler.cs

```diff
@@ -305,6 +305,18 @@ private IEnumerable<OngoingTask> CollectExternalReplicationTasks(List<ExternalRe
             return taskInfo;
         }
 
+        [RavenAction("/databases/*/admin/backup-task/delay", "POST", AuthorizationStatus.DatabaseAdmin)]
```

Comment:
wonder if this should be `Operator`, especially if we have server-wide backups

Reply:
Done.